Tools/scripts/AdditionsGenerator.sh

```diff
@@ -0,0 +1,618 @@
+                        cp -f "$sourceDataPath/mappostable.txt" "$destinationDataPath/"
+                        cp -f "$sourceDataPath/metalprocessitemtable.txt" "$destinationDataPath/"
+                        cp -f "$sourceDataPath/mp3nametable.txt" "$destinationDataPath/"
+                        cp -f "$sourceDataPath/num2cardillustnametable.txt" "$destinationDataPath/"
+                        cp -f "$sourceDataPath/resnametable.txt" "$destinationDataPath/"
+                        cp -f "$sourceDataPath/viewpointtable.txt" "$destinationDataPath/"
+                        for ((i=1; i<=17; i++)); do
+                            datac[$i]="[ Copied ]"
+                        done
+                        ;;
+                    2) cp -f "$sourceDataPath/bookitemnametable.txt" "$destinationDataPath/" && datac[2]="[ Copied ]" ;;
+                    3) cp -f "$sourceDataPath/buyingstoreitemlist.txt" "$destinationDataPath/" && datac[3]="[ Copied ]" ;;
+                    4) 
+                        cp -f "$sourceDataPath/carditemnametable.txt" "$destinationDataPath/"
+                        cp -f "$sourceDataPath/cardpostfixnametable.txt" "$destinationDataPath/"
+                        cp -f "$sourceDataPath/num2cardillustnametable.txt" "$destinationDataPath/"
+                        datac[4]="[ Copied ]"
+                        ;;
+                    5) cp -f "$sourceDataPath/etcinfo.txt" "$destinationDataPath/" && datac[5]="[ Copied ]" ;;
+                    6) cp -f "$sourceDataPath/exceptionminimapnametable.txt" "$destinationDataPath/" && datac[6]="[ Copied ]" ;;
+                    7) cp -f "$sourceDataPath/fogParameterTable.txt" "$destinationDataPath/" && datac[7]="[ Copied ]" ;;
+                    8) cp -f "$sourceDataPath/indoorrswtable.txt" "$destinationDataPath/" && datac[8]="[ Copied ]" ;;
+                    9) cp -f "$sourceDataPath/itemmoveinfov5.txt" "$destinationDataPath/" && datac[9]="[ Copied ]" ;;
+                    10) cp -f "$sourceDataPath/leveluseskillspamount.txt" "$destinationDataPath/" && datac[10]="[ Copied ]" ;;
+                    11) cp -f "$sourceDataPath/manner.txt" "$destinationDataPath/" && datac[11]="[ Copied ]" ;;
+                    12) cp -f "$sourceDataPath/mapobjlighttable.txt" "$destinationDataPath/" && datac[12]="[ Copied ]" ;;
+                    13) cp -f "$sourceDataPath/mappostable.txt" "$destinationDataPath/" && datac[13]="[ Copied ]" ;;
+                    14) cp -f "$sourceDataPath/metalprocessitemtable.txt" "$destinationDataPath/" && datac[14]="[ Copied ]" ;;
+                    15) cp -f "$sourceDataPath/mp3nametable.txt" "$destinationDataPath/" && datac[15]="[ Copied ]" ;;
+                    16) cp -f "$sourceDataPath/resnametable.txt" "$destinationDataPath/" && datac[16]="[ Copied ]" ;;
+                    17) cp -f "$sourceDataPath/viewpointtable.txt" "$destinationDataPath/" && datac[17]="[ Copied ]" ;;
+                esac
+            done
```

Comment:
Add a `read -p "Press enter to continue"` after each process, in case there are errors.

Reply:
Note: I'm just learning a bit about this review thingy, never really used it.
So don't take these too seriously, I will write comments first about it xD.

Reply:
That's fine! Thank you for your review, I will address the comments as soon as possible. :)